Documentation/document.css

```diff
@@ -43,14 +45,34 @@ aside{
 }
 
 main{
-    color: white;
-    background-color: turquoise;
-    flex-grow: 1;
+   
+    width: 100%;
+    height: auto;
+    /* color: rgb(134, 190, 223); */
 }
-.content{
+/* .content{
     
     background: rgb(200, 200, 200);
     height: 300px;
     padding: 20px;
     margin-bottom: 20px;
-}
\ No newline at end of file
+} */
+
+
+.doc-header{
+    padding-left: 2rem;
+    font-size: 2.5rem;
+    color: rgb(134, 190, 223); 
```

Comment:
use color variable

Reply:
done ,doc-txt-color variable used instead of color